### PR TITLE
Cat adoption page

### PR DESCRIPTION
**Pull request by TheaAleksovDev**
# Zero To Mastery - Webblocks 2022 📦📦

👉 _**Please remove the below and replace with your own values, leaving the headers where they are.**_ 👈

## PR contains:
Page that contains three cards with cats for adoption

## Breaking changes
None

## Screenshots
Screenshot 2022-10-18 155901

### CODE REVIEW

File: src/blocks/TheaAleksovDev4/script.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    container.innerHTML += `
+    <div class="animal">
+    <img class="image" src=${animalObj.image}>
+    <h1 class="name">${animalObj.name} (${animalObj.age} old)</h1>
+    <h2 class="contact"><i class="fa-solid fa-phone"></i> ${animalObj.contactPhone}</h2>
+    </div class="animal">`
+}
+
+
+addAdoption(new Animal("Lily","3 months","012-345678","./images/1.jpeg"))
```
Comment (sophiabrandt) [inline src/blocks/TheaAleksovDev4/script.js:20]: What are the licenses for the images? We need to make sure that we don't violate copyright.

Comment (TheaAleksovDev) [reply src/blocks/TheaAleksovDev4/script.js:20]: They are from Google. Can I replace them with some images from Pexels?

Comment (sophiabrandt) [reply src/blocks/TheaAleksovDev4/script.js:20]: Yes, that's a good idea.

**Comment by sophiabrandt** [comment]
Both #294 and #295 contain (partly) the same files, e.g., account.html. Please don't resubmit doubles, that makes it hard to review.

**Review by MattCSmith — APPROVED** [review]
Looks good to me 👍

**Pull request merged**